driver/bpf/fillers.h

```diff
@@ -738,6 +738,41 @@ static __always_inline int bpf_parse_readv_writev_bufs(struct filler_data *data,
 	return res;
 }
 
+FILLER(sys_readv_e, true)
+{
+	unsigned long val;
+	int32_t fd;
+	int res;
+
+	/*
+	 * fd
+	 */
+	val = bpf_syscall_get_argument(data, 0);
+	fd = (int32_t)val;
+	return bpf_val_to_ring(data, (int64_t)fd);
+}
+
+FILLER(sys_preadv_e, true)
```

Comment:
if we use just that filler for both 64 bit and 32 bit capture we need the 
```c
#ifndef CAPTURE_64BIT_ARGS_SINGLE_REGISTER
#error Implement this
#error Implement this
#endif
#endif
```

like in `sys_pread64_e`